### ISSUE TITLE
[Question] how to run some selective tests run on a single worker while others are running in parallel using multiple workers

### ISSUE THREAD

**Issue by lsingh23** [microsoft/playwright#15970]
Is there a way to make the tests work in parallel mode for some tests while the others in a serial mode. I know we can do something like separating test projects into two different projects and run each of them according to specific workers. Is this approach suitable for running tests on CI as well?

**Comment by pavelfeldman** [comment]
You can certainly split into projects and run some of them with --workers=1, that way tests work serially even without the describe.serial.

**Comment by pavelfeldman** [comment]
Closing as per above, please feel free to open a new issue if this does not cover your use case.
